src/Files.App/Helpers/Environment/SoftwareHelpers.cs

```diff
@@ -30,6 +31,32 @@ public static bool IsVSInstalled()
 			return true;
 		}
 
+		public static bool IsPythonInstalled()
```

Comment:
What happens when dragging a file if we don't have this check?

Reply:
The main scenario I considered is the following:

User **does not** have Python installed and this check **does not** exist.

This check exists to prevent a user without Python installed on their machine from seeing this prompt:
![image](https://github.com/files-community/Files/assets/28791910/ef9141fc-aad9-4799-831a-0e860cef1ed9)

WITHOUT checking that python is installed, a user will still see that prompt, and may cause confusion.
If this check is removed, differnt safeguards will need to be implemented to disallow the ```OpenItemsWithExecutableAsync(...)``` from being called with python files.

That being said, people with python files, will most likely already have python installed, so it probably is not an issue in the grand scheme of things.

Reply:
The question is if it's worth checking every time the user drags a file, it might be better to just show the tooltip and have nothing happen.

Reply:
Yes, that is understandable.
The other option is to add the check after the drop happens to prevent the call to Win32Helpers.InvokeWin32ComponentAsync(...), that way we avoid the performance hit for every drag and it only happens after the drop onto a python file.

Reply:
That would be good, we could also cache the result but I don't think we need to go that far.